### DISCUSSION TITLE
"Prefer not to say" on certain demographics questions

### DISCUSSION THREAD

**Discussion by LeaVerou** [Devographics/surveys#214] [General Survey Design]
This was pointed out by both a user study participant, and a pilot subject today. The intent was distinct from simply leaving the question blank, which didn't feel right to the participants ("I understand it's optional but I felt I owed an answer") — they wanted to communicate their intent to keep this data private, not be lumped together with people who simply ignored the question or never reached that section. 

If we do add it, it should probably come right before 🚫 None, and also include an emoji for visual anchoring (ideas: 🤐 🤫 🔒).

This came up in: 
- Income
- Gender
- Industry sector (though this may be an outlier – respondent couldn't find any preselected options that fit and didn't want to enter a custom one because they felt it identified them too much)

It's also common in disability and race.

**Comment by SachaG** [comment]
I think the right way to address that would be with a more general "skip" button on every question. 

I get that there's a slight difference between skipping because this doesn't apply to you and skipping because it does but you don't want to provide that data, but I think it's probably ok to treat them both as the same.

**Reply by LeaVerou** [reply]
This was not an ergonomics issue, they were not bothered by scrolling or anything. They knew they could skip the question. It just didn't communicate their intent. So I think that would add clutter without providing much value.
